proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,535 @@
+
+## 10. Protokolle
+
+Alle Beschlüsse des Vorstandes und der Mitgliederversammlung werden schriftlich
+in deutscher Sprache dokumentiert und stehen den Mitgliedern zur Einsichtnahme
+und zur Prüfung der Beschlüsse durch das Registergericht zur Verfügung.
+
+## 11. Internetabstimmung
+
+Die Vereinigung kann Abstimmungen über das Internet durchführen. Eine
+Internetabstimmung ist gültig nach einer Ankündigung per E-Mail oder im
+obligatorischen Gruppenchat der Kernmitglieder und wenn ein Quorum von
+mindestens 1/2 der Kernmitglieder teilnimmt. Der Abstimmungszeitraum muss
+mindestens zwei Wochen nach der Ankündigung betragen, um sicherzustellen,
+dass die Mitglieder genügend Zeit haben, ihre Stimme abzugeben.
+
+## 12. Tarifverträge
+
+Vor der Anstellung hauptamtlich Beschäftigter regelt der Vorstand deren
+Bezahlung in der Geschäftsordnung.
+
+## 13. Vereinsfinanzierung
+
+1. Der Verein wird durch Mitgliedsbeiträge, öffentliche Zuschüsse, Spenden
+   und sonstige Zuwendungen Dritter finanziert.
+2. Die Höhe der Mitgliedsbeiträge wird durch die Mitgliederversammlung oder
+   durch eine Internetabstimmung der Kernmitglieder entschieden.
+3. Im Falle der Auflösung oder Liquidation geht das gesamte Vermögen an eine
+   geeignete gemeinnützige Organisation für gemeinnützige Zwecke.
+
+## 14. Datenschutz im Verein
+
+1. Zur Erfüllung der Zwecke und Aufgaben des Vereins werden personenbezogene
+   Daten der Mitglieder unter Beachtung der Vorgaben der EU-Datenschutz-
+   Grundverordnung (DS-GVO) und des Bundesdatenschutzgesetzes (BDSG) verarbeitet.
+2. Den Organen des Vereins und sonstigen für den Verein tätigen Personen ist
+   es untersagt ohne Zustimmung, personenbezogene Daten oder private
+   Nachrichten zu einem anderen als zu dem zur Aufgabenerfüllung gehörenden
+   Zweck zu verarbeiten oder Dritten zugänglich zu machen. Diese Pflicht
+   besteht auch über das Ausscheiden der oben genannten Personen aus dem Verein
+   hinaus.
+3. Der Verein wahrt das Briefgeheimnis für den gesamten Schriftverkehr gemäß
+   Artikel 10 des deutschen Grundgesetzes und verpflichtet sich zur Einhaltung
+   und verpflichtet sich, soweit anwendbar, zu gleichwertigen Standards bei
+   internationaler Kommunikation.
+4. Der Verein respektiert die Privatsphäre der Nutzer der angebotenen
+   Web-Dienste und der veröffentlichten Software und erlaubt ihnen die anonyme
+   Teilnahme oder mit vollständiger Identifizierung.
+5. Die Vereinigung verpflichtet sich zur Transparenz und sammelt keine Daten
+   der Nutzer ohne deren ausdrückliche Kenntnisnahme und Zustimmung.
+6. In Fällen, in denen die Dienste des Vereins von einem Drittanbieter
+   bereitgestellt werden die eine separate Benutzeranmeldung erfordern, gelten
+   die Datenschutzbestimmungen des Drittanbieters.
+
+## 14. Salvatorische Klausel
+
+Sollten einzelne Bestimmungen dieser Satzung ganz oder teilweise unwirksam oder
+undurchführbar sein oder werden, so berührt dies nicht die Gültigkeit der
+Satzung insgesamt.
```

Comment:
Isn't this a clause for contracts? I don't think that this make sense in a statue. Where do we have this from?

Reply:
Some Satzungen you find online have that clause. It is probably redundant with the clause 8.9 where the board is allowed to change the Satzung.

Reply:
For such legal clauses I recommend to follow the Mustersatzung for the ministry of justice: https://www.bmj.de/SharedDocs/Downloads/DE/Formular/Mustersatzung_eines_Vereins.pdf?__blob=publicationFile&v=4